Click fails in command

I was trying to use `rich-click` instead of `click` in [jpterm](https://github.com/davidbrochart/jpterm/pull/17), but it fails with:
```python
  File "/home/david/mambaforge/envs/jpterm/bin/jpterm", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/home/david/github/davidbrochart/jpterm/jpterm/cli.py", line 16, in main
    command = click.command(_main)
              ^^^^^^^^^^^^^^^^^^^^
  File "/home/david/mambaforge/envs/jpterm/lib/python3.11/site-packages/rich_click/__init__.py", line 53, in command
    return click_command(*args, cls=cls, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/david/mambaforge/envs/jpterm/lib/python3.11/site-packages/click/decorators.py", line 190, in command
    assert cls is None, "Use 'command(cls=cls)(callable)' to specify a class."
AssertionError: Use 'command(cls=cls)(callable)' to specify a class.
```
I didn't investigate further, just posting to see if there is something obvious.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -4,6 +4,7 @@
 
 - Removed support for Typer - please use the [native functionality](https://typer.tiangolo.com/tutorial/options/help/#cli-options-help-panels) instead.
 - Added self-updating automated readme screengrabs using [rich-codex](https://github.com/ewels/rich-codex)
+- Fix click command call [#95](https://github.com/ewels/rich-click/pull/95)
 
 ## Version 1.5.2 (2022-08-01)
 

--- a/src/rich_click/__init__.py
+++ b/src/rich_click/__init__.py
@@ -50,4 +50,4 @@ def command(*args, cls=RichCommand, **kwargs):
 
     Defines the command() function so that it uses the RichCommand class by default.
     """
-    return click_command(*args, cls=cls, **kwargs)
+    return click_command(cls=cls)(*args, **kwargs)

